staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -51,8 +52,8 @@ type TransportConfig struct {
 	TracerProvider *trace.TracerProvider
 }
 
-// Config is configuration for creating a storage backend.
-type Config struct {
+// FactoryConfig is configuration of a storage factory
+type FactoryConfig struct {
```

Comment:
I'm personally not opposed to this name change, but I want someone from core api-machinery team to confirm it. The context is that we need GroupResourcee to be passed down with the config to 
 provide a reasonable metrics https://github.com/kubernetes/kubernetes/pull/104587

@deads2k @sttts - can you please confirm this change?

Reply:
The alternative is here: https://github.com/kubernetes/kubernetes/pull/104652

Reply:
Am not a fan of the "factory" keyword and the abstraction complexity this implies. Strong +1 for #104652 instead of this PR here.

Reply:
Thanks @sttts ! Let's close this one and proceed with the other one then.